assets/javascripts/omise-payment-form-handler.js

```diff
@@ -70,59 +70,61 @@
 					}
 				});
 
-				var omise_card_name   = $( '#omise_card_name' ).val(),
-					omise_card_number   = $( '#omise_card_number' ).val(),
-					omise_card_expiration_month   = $( '#omise_card_expiration_month' ).val(),
-					omise_card_expiration_year = $( '#omise_card_expiration_year' ).val(),
-					omise_card_security_code    = $( '#omise_card_security_code' ).val();
-				
-				// Serialize the card into a valid card object.
-				var card = {
-				    "name": omise_card_name,
-				    "number": omise_card_number,
-				    "expiration_month": omise_card_expiration_month,
-				    "expiration_year": omise_card_expiration_year,
-				    "security_code": omise_card_security_code
-				};
-				
-				var errors = OmiseUtil.validate_card(card);
-				if(errors.length > 0){
+				let errors            = [],
```

Comment:
This code is identical to that in the other file. Feels like maybe we should be reusing?

Reply:
Yup, I think so that we may somehow can merge these 2 JS files together. (however, please ignore the duplication here for a while as it is a bit out of scope of this PR)